### ISSUE TITLE
[Support]: Network Degradation Issue Potentially Linked to Frigate Add-on

### ISSUE THREAD

**Issue by Nevenit** [blakeblackshear/frigate-hass-addons#186]
Hi everyone,

I've been experiencing a recurring issue where my entire network becomes unusable once or twice a week. When this happens, ping times to 1.1.1.1 shoot up to around 4 seconds, and the network is incredibly unstable. This can last anywhere from a few minutes to a few hours. The problem has been occurring for several months, and I initially struggled to pinpoint the cause.

Recently, I discovered that stopping the Frigate add-on in Home Assistant seems to immediately resolve the issue. However, I haven't been able to find anything unusual in the Frigate logs that would indicate a problem. The issue affects all devices on my network, including those connected via Ethernet, not just WiFi devices.

Here are some details about my setup:

-Frigate is running as an add-on in Home Assistant.
-I have 3 WiFi cameras and 1 PoE camera connected to Frigate.
-Network instability occurs at random times, without any clear triggers.
-Stopping the Frigate add-on resolves the issue, but restarting it sometimes causes the problem to reappear.
-At first, I suspected the issue was caused by remapping the Home Assistant media folder onto a network drive, but switching it back to a local folder did not resolve the problem.
I'll also include some images of the network performance to illustrate the issue.

I'm not sure if this is a Frigate-specific issue or if something else in my setup is causing it, but Frigate seems to be a common factor. I'd really appreciate any guidance on what might be causing this or how I could further troubleshoot the problem.

Thanks in advance for your help!

![brave_AdfYhjH7Hu](https://github.com/user-attachments/assets/f00d16f0-0ecb-4d68-9bae-886d06d25deb)
![brave_4YLXVojKDP](https://github.com/user-attachments/assets/a262b352-a1c0-4cd9-9b93-a590678dcce9)
![WindowsTerminal_U1Kp8MLOmq](https://github.com/user-attachments/assets/1a934349-f299-4786-a939-aa4fb265d7e8)
![brave_5LoCPIPB5v](https://github.com/user-attachments/assets/505602b8-e6ea-46e1-b041-a5a6bc1879da)

**Comment by NickM-27** [comment]
Frigate is always streaming data from cameras so it may be some issue there, I don't see how frigate would be doing anything specific to have the switch to have problems

**Comment by Nevenit** [comment]
> Frigate is always streaming data from cameras so it may be some issue there, I don't see how frigate would be doing anything specific to have the switch to have problems

But if its always doing it then wouldn't the issue be consistently always happening? Its very random. I tried looking at wireshark while it happens and i dont see anything weird. A lot more packets fail but thats just cuz the internet gets so bad

**Comment by NickM-27** [comment]
not necessarily, could be caused by any bug in the switch firmware or anything else. Or perhaps only in certain circumstances. Would be good to try and get logs from other parts of the stack

**Comment by Nevenit** [comment]
There is nothing. Its been months. This issue is actually wearing me down. Out of nowhere my network becomes unusable, i stop the frigate addon and it goes back to normal, i restart it it goes to shit again. There are no logs anywhere, router is fine, frigate and home assistant are fine, nothing weird happening in wire shark. I am losing my mind.

**Comment by 3dfxgamer** [comment]
I'm having the same issue, although in my case it was constant, but it appeared out of nowhere. I spent many days trying to fix it, resetting configurations, switches, and searching for AI solutions, etc. By chance, I stopped the Frigate Docker container and it was resolved.
I have 2 Deco xE75s and 2 Deco S7s connected via a single SG2218 switch. The logs didn't show anything that would indicate it was a Frigate problem. When Frigate is running, the cameras and other devices experience constant disconnections, ping drops, and lag.

**Comment by Nevenit** [comment]
I did solve my issue, it wasn't frigate or anything software related. It was the damn network card in the machine that was running frigate. To this day i don't know what was happening or how but all i did was put in a new network card in and the issue went away.

**Comment by 3dfxgamer** [comment]
Thanks for the update.
Interesting, in my case I discovered that a specific Ezviz camera (I have 5 in total) was the one causing the problem. Frigate had that camera added as a stream, and whether I accessed it via Frigate or the Ezviz app itself, the whole network would crash. I removed that camera from the configuration and everything returned to normal. I suspect it was a firmware that must have updated automatically on that camera.

**Comment by Nevenit** [comment]
Im amazed that a single faulty device can take down your entire network...
